### ISSUE TITLE
unicode support for headers

### ISSUE THREAD

**Issue by tarsisazevedo** [airbnb/knowledge-repo#261]
Auto-reviewers: @NiharikaRay @matthewwardrop @earthmancash @danfrankj

It's not possible to have unicode chars in headers. It's important to have it for non-english posts.

stacktrace:
```bash
(knowledge-repo) tarsis@fury-road [/.../projetos/bigdata/bigdata-knowledge-repo]  (master)
$ knowledge_repo --repo posts/ add globoesporte/impacto_tr.kp/knowledge.md -p esportes --update
Traceback (most recent call last):
  File "/Users/tarsis/.virtualenvs/knowledge-repo/bin/knowledge_repo", line 255, in <module>
    kp = knowledge_repo.KnowledgePost.from_file(args.filename, src_paths=args.src)
  File "/Users/tarsis/.virtualenvs/knowledge-repo/lib/python2.7/site-packages/knowledge_repo/post.py", line 350, in from_file
    kp = KnowledgePostConverter.for_file(cls(), filename, format=format, postprocessors=postprocessors).from_file(filename, **opts)
  File "/Users/tarsis/.virtualenvs/knowledge-repo/lib/python2.7/site-packages/knowledge_repo/converter.py", line 53, in wrapped
    KnowledgePostProcessor._get_subclass_for(postprocessor).process(kp)
  File "/Users/tarsis/.virtualenvs/knowledge-repo/lib/python2.7/site-packages/knowledge_repo/postprocessors/format_checks.py", line 33, in process
    field, type(headers[field]), typ)
AssertionError: Value for field `title` is of type <type 'unicode'>, and needs to be of type <type 'str'>.
```

**Comment by matthewwardrop** [comment]
Thanks for the issue report! I suspect this is simply due to over-zealous format checking. Will submit a fix in the next day or so.

**Comment by matthewwardrop** [comment]
This is fixed in #293, which should be merged shortly. See attached screenshot for proof :).
<img width="966" alt="screen shot 2017-07-03 at 6 23 49 pm" src="https://user-images.githubusercontent.com/124910/27811642-773c2066-601d-11e7-9699-29e63f20a49d.png">
